lib/less/parser/parser.js

```diff
@@ -421,7 +421,11 @@ var Parser = function Parser(context, imports, fileInfo) {
                     }
 
                     parserInput.forget();
-                    return new(tree.Call)(name, args, index, fileInfo);
+                    var call = new(tree.Call)(name, args, index, fileInfo);
+                    if (name === 'calc') {
```

Comment:
Should not this code (i.e. testing if the func name is `calc` and setting  temporary math-on for args) be paced right into `Call.prototype.eval` below?

Reply:
That's a good point, let me see if I can simplify it.